package.json

```diff
@@ -36,6 +36,7 @@
     "npm": ">=6.4.0"
   },
   "dependencies": {
+    "got": "11.8.5",
     "guid-typescript": "^1.0.9",
     "jsonwebtoken": "^8.5.1",
     "request": "^2.88.2",
```

Comment:
```suggestion

```

I guess we should remove the dependency since we are replacing it. Some for `request-promise`.